src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -199,30 +193,28 @@
                                                  (filter #(not (valid-networks-set %))
                                                          not-available-networks)))
                                        (distinct))]
-    (cond-> (->> valid-networks
-                 (map
-                  (fn [chain-id]
-                    (cond->
-                      {:chain-id chain-id
-                       :type     (cond
-                                   (contains? not-available-networks-set chain-id)         :not-available
-                                   (or receiver?
-                                       (not (contains? disabled-set chain-id)))            :loading
-                                   (and (not receiver?) (contains? disabled-set chain-id)) :disabled)}
-                      (and (not receiver?) (contains? disabled-set chain-id))
-                      (assoc :total-amount (money/bignumber "0")))))
-                 (filter
-                  (fn [network-amount]
-                    (or (and receiver?
-                             (or (= tx-type :tx/bridge)
-                                 (contains? receiver-networks-set (:chain-id network-amount))))
-                        (not receiver?))))
-                 (sort-by (fn [network-amount]
-                            (get network-priority-score
-                                 (network-utils/id->network (:chain-id network-amount)))))
-                 (vec))
-      (and receiver? (< receiver-networks-count available-networks-count) (not= tx-type :tx/bridge))
-      (conj {:type :add}))))
```

Comment:
We don't need to add this map anymore, so removing the `(cond->` statement. No other relevant changes here.